src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -50,6 +51,9 @@
 namespace osvr {
 namespace pluginhost {
     static const auto PLUGIN_HOST_LOGGER_NAME = "PluginHost";
+#ifdef _MSC_VER
+    static const auto PLUGIN_HOST_DEBUG_SUFFIX = ".debug";
```

Comment:
because we don't need to filter plugins on other platforms, only on MSVC because it has two different C runtimes?